bricks-maven-plugin/src/test/java/com/github/ingogriebsch/bricks/maven/plugin/analyzer/discovery/AnalyzersTest.java

```diff
@@ -35,79 +44,92 @@
 
 import lombok.RequiredArgsConstructor;
 
-@SuppressWarnings("unchecked")
 public class AnalyzersTest {
 
     @Test
     public void testOK() throws Exception {
-        AnalyzerDiscovery testDiscovery = x -> Lists.newArrayList(DummyAnalyzer.of(AnalysisResult.OK));
+        AnalyzerDiscovery testDiscovery = x -> newArrayList(DummyAnalyzer.of(OK));
```

Comment:
no strong option, but here for example, you could argue while being more concise, your version puts mor cognitive load on the reader.

Reply:
for Assertions.* it is obvious. Maybe Collector.*, too... 
For the first, the reason is the context and the uniform naming of the methods, togehther with the need to use many of those calls in one CompilationUnit.
For Collectors.*, i think they are very well known and very rarely extended or replaced.

For project classes, like AnalyzerResult.OK, i am less sure. as said, it makes the code harder to comprehend when you approach it the first time. but no strong opinion.